backend/migrations/001_initial.sql

```diff
@@ -0,0 +1,146 @@
+-- +goose Up
+
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS Organisations (
+    id INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(150),
+    email VARCHAR(150) NOT NULL,
+    githubID VARCHAR(150) NOT NULL,
+    avatarUrl VARCHAR(300),
+    webSite VARCHAR(300),
+    `description` VARCHAR(300),
+
+    createdAt TIMESTAMP,
+    updatedAt TIMESTAMP,
+    deletedAt TIMESTAMP,
+
+    UNIQUE(email, githubID),
+    PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Users (
+    id INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(150),
+    email VARCHAR(150) NOT NULL,
+    githubID VARCHAR(150) NOT NULL,
+    avatarUrl VARCHAR(300),
+
+    createdAt TIMESTAMP,
+    updatedAt TIMESTAMP,
+    deletedAt TIMESTAMP,
+
+    UNIQUE(email, githubID),
+    PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Wallets (
+    id INT NOT NULL AUTO_INCREMENT,
+
+    walletID VARCHAR(300) NOT NULL,
+    userID   INT NOT NULL,
+    organisationID   INT NOT NULL,
+    balance INT NOT NULL,
+
+    createdAt TIMESTAMP,
+    updatedAt TIMESTAMP,
+    deletedAt TIMESTAMP,
+
+    UNIQUE(walletID),
+    PRIMARY KEY(Id),
+    FOREIGN KEY (userID) REFERENCES Users(userID),
+    FOREIGN KEY (organisationID) REFERENCES Organisations(organisationID)
+);
+
+CREATE TABLE IF NOT EXISTS Articles (
+    id INT NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(150),
+    `description` TEXT,
+    quantity INTEGER,
+    yotas INT NOT NULL,
```

Comment:
I think for semantic it is better to rename this field price instead of yotas, because yotas is the currency

Reply:
ok, i will change that !